versions.yaml

```diff
@@ -186,7 +186,7 @@ externals:
     description: |
       OCI-based Kubernetes Container Runtime Interface implementation
     url: "https://github.com/cri-o/cri-o"
-    version: "v1.13.0"
+    version: "v1.13.6"
```

Comment:
we should use cri-o version 1.14.x for k8s 1.14.x as stated in the [cri-o compatibility matrix](https://github.com/cri-o/cri-o#compatibility-matrix-cri-o---kubernetes-clusters)

Reply:
@chavafg currently cri-o version 1.14.x is not working properly as I talked with @ganeshmaharaj so that is the reason I am using 1.13.6

Reply:
what errors are you getting?

Reply:
@chavafg , here are the errors
```
$ make crio
bash .ci/install_bats.sh
/usr/bin/bats
RUNTIME=kata-runtime ./integration/cri-o/cri-o.sh
/home/gabyubuntu/go/src/github.com/cri-o/cri-o /home/gabyubuntu/go/src/github.com/kata-containers/tests
HEAD is now at 0b8a2dff version: v1.14.0
/home/gabyubuntu/go/src/github.com/kata-containers/tests
/home/gabyubuntu/go/src/github.com/cri-o/cri-o/test /home/gabyubuntu/go/src/github.com/kata-containers/tests
fdisk: cannot open /dev/vdb: No such file or directory
Ensure crio service is stopped before running the tests
Ensure docker service is stopped before running the tests
++ time bats --tap ctr.bats
1..40
not ok 1 ctr not found correct error message
# (from function `setup_crio' in file helpers.bash, line 229,
#  from function `start_crio' in file helpers.bash, line 286,
#  in test file ctr.bats, line 10)
#   `start_crio' failed
# time="2019-05-06T20:01:22Z" level=error msg="error opening storage: /dev/vdb is not available for use with devicemapper"
# time="2019-05-06T20:01:24Z" level=fatal msg="failed to connect: failed to connect: context deadline exceeded"
# time="2019-05-06T20:01:26Z" level=fatal msg="failed to connect: failed to connect: context deadline exceeded"
# 0
# File descriptor 3 (pipe:[12816053]) leaked on lvm invocation. Parent PID 12616: bash
#   Volume group "storage" not found
#   Cannot process volume group storage
# File descriptor 3 (pipe:[12816053]) leaked on lvm invocation. Parent PID 12616: bash
#   Volume group "storage" not found
#   Cannot process volume group storage
# File descriptor 3 (pipe:[12816053]) leaked on lvm invocation. Parent PID 12616: bash
#   Device /dev/vdb not found.
not ok 2 ctr termination reason Completed
# (from function `setup_crio' in file helpers.bash, line 229,
#  from function `start_crio' in file helpers.bash, line 286,
#  in test file ctr.bats, line 19)
#   `start_crio' failed
# time="2019-05-06T20:01:26Z" level=error msg="error opening storage: /dev/vdb is not available for use with devicemapper"
# time="2019-05-06T20:01:28Z" level=fatal msg="failed to connect: failed to connect: context deadline exceeded"
# time="2019-05-06T20:01:30Z" level=fatal msg="failed to connect: failed to connect: context deadline exceeded"
# 0
# File descriptor 3 (pipe:[12816053]) leaked on lvm invocation. Parent PID 12762: bash
#   Volume group "storage" not found
#   Cannot process volume group storage
# File descriptor 3 (pipe:[12816053]) leaked on lvm invocation. Parent PID 12762: bash
#   Volume group "storage" not found
#   Cannot process volume group storage
# File descriptor 3 (pipe:[12816053]) leaked on lvm invocation. Parent PID 12762: bash
#   Device /dev/vdb not found.
not ok 3 ctr termination reason Error
# (from function `setup_crio' in file helpers.bash, line 229,
#  from function `start_crio' in file helpers.bash, line 286,
#  in test file ctr.bats, line 44)
#   `start_crio' failed
# time="2019-05-06T20:01:30Z" level=error msg="error opening storage: /dev/vdb is not available for use with devicemapper"
# time="2019-05-06T20:01:32Z" level=fatal msg="failed to connect: failed to connect: context deadline exceeded"
# time="2019-05-06T20:01:34Z" level=fatal msg="failed to connect: failed to connect: context deadline exceeded"
# 0
# File descriptor 3 (pipe:[12816053]) leaked on lvm invocation. Parent PID 12905: bash
#   Volume group "storage" not found
#   Cannot process volume group storage
# File descriptor 3 (pipe:[12816053]) leaked on lvm invocation. Parent PID 12905: bash
#   Volume group "storage" not found
#   Cannot process volume group storage
# File descriptor 3 (pipe:[12816053]) leaked on lvm invocation. Parent PID 12905: bash
#   Device /dev/vdb not found.
not ok 4 ulimits
# (from function `setup_crio' in file helpers.bash, line 229,
#  from function `start_crio' in file helpers.bash, line 286,
#  in test file ctr.bats, line 71)
#   `ULIMITS="--default-ulimits nofile=42:42 --default-ulimits nproc=1024:2048" start_crio' failed
# time="2019-05-06T20:01:34Z" level=error msg="error opening storage: /dev/vdb is not available for use with devicemapper"
# time="2019-05-06T20:01:36Z" level=fatal msg="failed to connect: failed to connect: context deadline exceeded"
# time="2019-05-06T20:01:38Z" level=fatal msg="failed to connect: failed to connect: context deadline exceeded"
# 0
# File descriptor 3 (pipe:[12816053]) leaked on lvm invocation. Parent PID 13052: bash
#   Volume group "storage" not found
#   Cannot process volume group storage
# File descriptor 3 (pipe:[12816053]) leaked on lvm invocation. Parent PID 13052: bash
#   Volume group "storage" not found
#   Cannot process volume group storage
# File descriptor 3 (pipe:[12816053]) leaked on lvm invocation. Parent PID 13052: bash
#   Device /dev/vdb not found.
```